WcaOnRails/app/assets/stylesheets/relations.scss

```diff
@@ -0,0 +1,53 @@
+.slick-arrow {
+  &::before {
+    color: black;
+  }
+}
+
+.slick-slider {
+  margin-bottom: 0;
+}
+
+.relations-form {
+  .selectize-input {
+    width: 300px;
+  }
+
+  .button-box {
+    margin-top: 10px;
+  }
+}
+
+.relations-line {
+  margin-top: 40px;
+}
+
+.competition-box,
+.person-box {
+  margin: {
+    top: 10px;
+    bottom: 10px;
+  }
+  padding: 10px;
+  text-align: center;
+  border-radius: 10px;
+  text-align: center;
+  a {
+    color: white;
+    font-size: 2em;
+    text-decoration: none;
+  }
+}
+
+.person-box {
+  background-color: lightgray;
+}
+
+.competition-box {
+  background-color: $blue;
+}
+
+.description {
```

Comment:
This feels a bit general to me. I could see us accidentallly having another `div.description` somewhere in our codebase and we wouldn't want these rules to apply. Maybe scope this entire scss file inside of something?